### PR TITLE
UI Rework

### PR DESCRIPTION
**Pull request by 0xMRTT**

### CODE REVIEW

**Comment by 0xMRTT** [comment]
Screenshot from 2022-08-10 19-12-49
Screenshot from 2022-08-10 19-12-54
Screenshot from 2022-08-10 19-12-58
Screenshot from 2022-08-10 19-13-04
Screenshot from 2022-08-10 19-13-17
Screenshot from 2022-08-10 19-13-42
Screenshot from 2022-08-10 19-13-52

**Comment by daudix-UFO** [comment]
Things that I noticed is monochrome disk-symbolic and adw-gtk3-warning, is that on purpose?

**Comment by daudix-UFO** [comment]
And advanced tab don't have icon

image

**Comment by 0xMRTT** [comment]
> And advanced tab don't have icon
> 
> image

I use the integrated icons...

**Comment by 0xMRTT** [comment]
> Things that I noticed is monochrome disk-symbolic and adw-gtk3-warning, is that on purpose?

It's WIP ...
It's for the integration with plugins

**Comment by 0xMRTT** [comment]
Just need to finish some little things and after merge this PR.

Should we open an issue in TWIG (https://this week.gnome.org)? 

The welcome screen, preset manager and plugins support will come later.

**Comment by daudix-UFO** [comment]
I think we can!

**Comment by 0xMRTT** [comment]
> I think we can!

Join the matrix channel thisweek:gnome.org

**Comment by daudix-UFO** [comment]
Perhaps we will even have time to be in next TWIG!

**Comment by 0xMRTT** [comment]
Twig for this week close tomorrow at 20 UTC. So we have 29 hours left 😎

**Comment by daudix-UFO** [comment]
Maked disscussion about it #102

**Comment by daudix-UFO** [comment]
Can we move hamburger menu to make it first (like in other apps)?

**Comment by 0xMRTT** [comment]
Just next to the close button ?

**Comment by daudix-UFO** [comment]
Yes

**Comment by 0xMRTT** [comment]
Done

**Comment by 0xMRTT** [comment]
Finally. UI Rework done. 

Know issues:
- Adwaita Manager can freeze while generating a theme or it's slow. In the next release, there will be an AdwStatusPage showing the progress and the logs

If you found bugs. Comment here!

**Comment by ArtyIF** [comment]
Any final screenshots of the new interface?

**Comment by daudix-UFO** [comment]
image
image
image
image

File: src/window.py [hunk 1]
```diff
@@ -26,40 +26,159 @@
...
+        for v in store_values:
+            store.append(v)
+        self.tone_row.set_model(store)
+        self.monet_pref_group.add(self.tone_row)
+
+        self.monet_theme_row = Adw.ComboRow()
+        self.monet_theme_row.set_title(_("Theme"))
+
+        store = Gtk.StringList()
+        store.append("Automatic")
```
Comment (daudix-UFO) [inline src/window.py:152]: Maybe just "Auto"?

File: com.github.AdwCustomizerTeam.AdwCustomizer.json [hunk 2]
```diff
@@ -57,7 +71,7 @@
                 {
                     "type" : "git",
                     "path" : ".",
-                    "branch": "main"
+                    "branch": "next"
```
Comment (daudix-UFO) [inline com.github.AdwCustomizerTeam.AdwCustomizer.json:74]: Switch back to `main`

**Review by daudix-UFO — APPROVED** [review]

**Comment by ArtyIF** [comment]
Nice. I'd say:
- reverse the order of buttons in the title bar to the right - usually GTK apps put the menu button last
- don't forget to update the potfile - call `./po/update_potfile.sh` from the project's root
- speaking of which, replace `MIT/X11 license` with `GNU General Public License version 3`
- and update [this line](https://github.com/AdwCustomizerTeam/AdwCustomizer/blob/main/src/main.py#L454)

**Review by ArtyIF — CHANGES_REQUESTED** [review]
- [x] reverse the order of buttons in the title bar to the right - usually GTK apps put the menu button last
- [x] don't forget to update the potfile - call ./po/update_potfile.sh from the project's root
- [ ] speaking of which, replace MIT/X11 license with GNU General Public License version 3
- [x] replace any occurence of MIT or similar (you can open the bottom panel and click on the magnifying glass in GNOME Builder to search in the whole project)

**Comment by 0xMRTT** [comment]
> - [ ] don't forget to update the potfile - call ./po/update_potfile.sh from the project's root

@daudix-UFO It's maybe why locales doesn.t work

**Comment by 0xMRTT** [comment]
Started writing the Changelog and release notes. Currently saved as draft. (Go on release tab)

**Comment by daudix-UFO** [comment]
> Started writing the Changelog and release notes. Currently saved as draft. (Go on release tab)

@0xMRTT Polished a little ;)

**Comment by daudix-UFO** [comment]
Also, don't forget:

- reverse the order of buttons in the title bar to the right

**Comment by 0xMRTT** [comment]
It's already done.

The menu button is next to the close button.

**Comment by ArtyIF** [comment]
Was the potfile and the potfile script updated? I don't see it in any of the commits.

**Comment by 0xMRTT** [comment]
> Was the potfile and the potfile script updated? I don't see it in any of the commits.

Yes, didn't run yet

**Comment by 0xMRTT** [comment]
Screenshot from 2022-08-12 11-58-21
Screenshot from 2022-08-12 11-58-34
Screenshot from 2022-08-12 11-58-38
Screenshot from 2022-08-12 11-58-44

**Comment by 0xMRTT** [comment]
@ArtyIF @daudix-UFO @AndroGR. Can I merge ?

**Comment by daudix-UFO** [comment]
I think it works good

**Comment by 0xMRTT** [comment]
@all-contributors please add @0xMRTT for code

**Comment by 0xMRTT** [comment]
@all-contributors please add @daudix-UFO for design

**Comment by 0xMRTT** [comment]
@all-contributors please add @daudix-UFO for design

**Pull request merged**